apps/gdalalg_raster_pipeline.cpp

```diff
@@ -589,17 +606,20 @@ std::string GDALRasterPipelineAlgorithm::GetUsageForCLI(
         auto alg = GetStepAlg(name);
         assert(alg);
         if (!alg->CanBeFirstStep() && !alg->IsHidden() &&
-            name != GDALRasterWriteAlgorithm::NAME)
+            name != GDALRasterWriteAlgorithm::NAME &&
+            name != GDALRasterInfoAlgorithm::NAME)
         {
             ret += '\n';
             alg->SetCallPath({name});
             ret += alg->GetUsageForCLI(shortUsage, stepUsageOptions);
         }
     }
+    for (const std::string &name :
+         {GDALRasterInfoAlgorithm::NAME, GDALRasterWriteAlgorithm::NAME})
```

Comment:
CI is not happy about this, perhaps you could try
```suggestion
         {&GDALRasterInfoAlgorithm::NAME, &GDALRasterWriteAlgorithm::NAME})
```

Reply:
actually the type is "const char*". Changed to it